KeenClient/KeenClient.m

```diff
@@ -239,27 +231,7 @@ + (KIODBStore *) getDBStore {
 }
 
 - (id)init {
-    self = [super init];
-    
-    // log the current version number
-    if ([KeenClient isLoggingEnabled]) {
-        KCLog(@"KeenClient-iOS %@", kKeenSdkVersion);
-    }
-    
-    [self refreshCurrentLocation];
-    
-    self.uploadQueue = dispatch_queue_create("io.keen.uploader", DISPATCH_QUEUE_SERIAL);
-
-    // use global concurrent dispatch queue to run queries in parallel
-    self.queryQueue = dispatch_queue_create(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0);
-
-    self.maxEventUploadAttempts = 3;
-    
-    self.maxQueryAttempts = 10;
-    
-    self.queryTTL = 3600;
-
-    return self;
+    return [self initWithProjectID:@"pi" andWriteKey:@"wk" andReadKey:@"rk"];
```

Comment:
I'm not sure we want to initialize with default values like this, it could lead to confusion. Might be better to initialize with empty values and verify in `initWithProjectID` that they're not empty.